src/components/PlanCard/index.tsx

```diff
@@ -0,0 +1,75 @@
+import { Button } from 'components/Button'
+import { formatPrice } from 'utils/formatPrice'
+import {
+  ButtonContainer,
+  Container,
+  ContentContainer,
+  MinorPlanType,
+  OfferStyle,
+  PlanTypeStyle,
+  PriceContainer,
+  PriceContentContainer,
+  PriceStyle,
+  RenewStyles,
+  SignStyle,
+} from './styles'
```

Comment:
Adicionar quebras de linha
```suggestion
import { Button } from 'components/Button'

import { formatPrice } from 'utils/formatPrice'

import {
  ButtonContainer,
  Container,
  ContentContainer,
  MinorPlanType,
  OfferStyle,
  PlanTypeStyle,
  PriceContainer,
  PriceContentContainer,
  PriceStyle,
  RenewStyles,
  SignStyle,
} from './styles'
```